### ISSUE TITLE
Long range TK with binoculars trick

### ISSUE THREAD

**Issue by ATH1909** [tgstation/tgstation#48966]
## Round ID:
Not a /tg/ round (I found this trick on fulpstation, but it should still work over on /tg/).

## Testmerges:
Unknown; probably not relevant.

## Reproduction:

Use telekinesis to pick up binoculars from a distance, then press "z" with them in (telekinetic) hand to use the binoculars to expand your vision range (until you move)... even though that wouldn't logically work.

But wait, there's more! If you, without leaving your tile, then stop telekinetically holding the binoculars (or alternatively switch to your (occupied) off hand and press q), your other hand will still be occupied, but you'll have a free hand and keep the expanded vision range until you leave your tile. You can then use your free hand to use telekinesis on things that you can only see with this expanded vision range, which can be quite useful for telegriefing people from maintenance over a screen away. Note that there _is_ a maximum range for telekinetically gripping an object, it just happens to be longer than your normal viewing distance. Note that you _might_ have to stand on the same tile as your telekinetically held binoculars for this trick to work; I forgot the specifics.

tldr;
1. use TK on binoculars
2. use them in hand
3. drop them
4. keep your expanded vision range (until you move) and use it to perform long-range TK(+x-ray+thermal vision) shenanigans
5. also use it to deal some damage to a space dragon while remaining safe and completely hidden from it in maint
